README.md

```diff
@@ -118,6 +119,49 @@ goldrush run reads=long_reads G=3e9
 
 **For more information about the GoldRush algorithm and tips for running GoldRush see our [wiki](https://github.com/bcgsc/goldrush/wiki)** 
 
+# System Requirements
+
+## Hardware Requirements
+
+GoldRush requires a standard computer with enough RAM to support the operations defined by the user. For the assembly of a human genome with a ~60X coverage, GoldRush requires a computer with:
+
+RAM: 64+ GB 

```

Comment:
I'd consider just making these lines giving RAM/threads suggestions as part of your phrase instead of on separate lines? It feels a bit choppy maybe?

Reply:
one is a requirement, the other is a recommendation

Reply:
Right, but I meant that you can make this into a sentence - I don't see the point of breaking the lines.

Reply:
i see